### PR TITLE
block: Update {{uw-ublock-famous}} reason label

### PR DESCRIPTION
**Pull request by Ammarpad**
Changed "famous" to "well-known" per feedback on wiki. 

[Wikipedia talk:Twinkle#minor tweak to block reasons](https://en.wikipedia.org/w/index.php?title=Wikipedia_talk:Twinkle&oldid=930936807#minor_tweak_to_block_reasons)

### CODE REVIEW

**Comment by Ammarpad** [comment]
Yes, I note that inconsistency, but the reason I left it at that is that someone has to do the other part independently, (that's proposing the move itself). Now, that you raise the issue I will open move request for the template next time I log in.

**Comment by Amorymeltzer** [comment]
No worries!  I didn't mean to imply you *should* or anything, just that
it'd be good to let things play out further before committing.  Nothing
wrong with the PR at all.

>

**Comment by Amorymeltzer** [comment]
Noting here that a move discussion was indeed opened by @Ammarpad using my suggestion: https://en.wikipedia.org/wiki/Wikipedia_talk:Template_messages/User_talk_namespace#Requested_move_17_December_2019 and is trending towards a rename.

File: modules/twinkleblock.js [hunk 1]
```diff
@@ -785,7 +785,7 @@ Twinkle.block.blockPresetsInfo = {
 	'uw-ublock-famous': {
 		expiry: 'infinity',
 		forRegisteredOnly: true,
-		reason: '{{uw-ublock-famous}} <!-- Username represents a famous person, soft block -->',
+		reason: '{{uw-ublock-famous}} <!-- Username represents a well-known person, soft block -->',
```
Comment (Amorymeltzer) [inline modules/twinkleblock.js]:
```suggestion
		reason: '{{uw-ublock-wellknown}} <!-- Username represents a well-known person, soft block -->',
```

File: modules/twinkleblock.js [hunk 2]
```diff
@@ -907,7 +907,7 @@ Twinkle.block.blockGroups = [
 			{ label: 'Username violation, soft block', value: 'uw-ublock' },
 			{ label: 'Username violation, hard block', value: 'uw-uhblock' },
 			{ label: 'Username impersonation hard block', value: 'uw-uhblock-double' },
-			{ label: 'Username represents a famous person, soft block', value: 'uw-ublock-famous' },
+			{ label: 'Username represents a well-known person, soft block', value: 'uw-ublock-famous' },
```
Comment (Amorymeltzer) [inline modules/twinkleblock.js]:
```suggestion
			{ label: 'Username represents a well-known person, soft block', value: 'uw-ublock-wellknown' },
```

File: modules/twinkleblock.js [hunk 3]
```diff
@@ -785,7 +785,7 @@ Twinkle.block.blockPresetsInfo = {
 	'uw-ublock-famous': {
```
Comment (Amorymeltzer) [inline modules/twinkleblock.js]:
```suggestion
	'uw-ublock-wellknown': {
```

**Pull request merged**